test/runtests.jl

```diff
@@ -19,7 +19,9 @@ using Base: operator_associativity
 @time include("Meshes/cubedsphere.jl")
 
 @time include("Topologies/rectangle.jl")
+@time include("Topologies/rectangle_sfc.jl")
 @time include("Topologies/cubedsphere.jl")
+@time include("Topologies/cubedsphere_sfc.jl")
```

Comment:
Could you add a couple of unit tests that checks the necessary properties of `spacefillingcurve` and `linearindices` for both rectangle and cubed sphere meshes:

- that `spacefillingcurve(mesh)` actually includes all elements (sort and compare with `elements(mesh)`)
- that `linearindices` is actually the inverse (iterate through all elements, and check that it gives the correct values)

(you shouldn't need to construct a topology for these)